docs/services/bitcoind.md

```diff
@@ -1,129 +1,316 @@
 # Bitcoin Service
-The Bitcoin Service adds a native [Node.js](https://nodejs.org) interface to [Bitcoin Core](https://github.com/bitcoin/bitcoin) for querying information about the Bitcoin blockchain. Bindings are linked to Bitcoin Core compiled as a static library.
+
+The Bitcoin Service is a Node.js interface to [Bitcoin Core](https://github.com/bitcoin/bitcoin) for querying information about the bitcoin block chain. It will manage starting and stopping `bitcoind` or connect to several running `bitcoind` processes. It uses a branch of a [branch of Bitcoin Core](https://github.com/bitpay/bitcoin/tree/0.12-bitcore) with additional indexes for querying information about addresses and blocks. Results are cached for performance and there are several additional API methods added for common queries.
+
+## Configuration
+
+The default configuration will include a "spawn" configuration in "bitcoind". This defines the location of the block chain database and the location of the `bitcoind` daemon executable. The below configuration points to a local clone of `bitcoin`, and will start `bitcoind` automatically with your Node.js application.
+
+```json
+  "servicesConfig": {
+    "bitcoind": {
+      "spawn": {
+        "datadir": "/home/bitcore/.bitcoin",
+        "exec": "/home/bitcore/bitcoin/src/bitcoind"
+      }
+    }
+  }
+```
+
+It's also possible to connect to separately managed `bitcoind` processes with round-robin quering, for example:
+
+```json
+  "servicesConfig": {
+    "bitcoind": {
+      "connect": [
+        {
+          "rpchost": "127.0.0.1",
+          "rpcport": 30521,
+          "rpcuser": "bitcoin",
+          "rpcpassword": "local321",
+          "zmqpubrawtx": "tcp://127.0.0.1:30611"
+        },
+        {
+          "rpchost": "127.0.0.1",
+          "rpcport": 30522,
+          "rpcuser": "bitcoin",
+          "rpcpassword": "local321",
+          "zmqpubrawtx": "tcp://127.0.0.1:30622"
+        },
+        {
+          "rpchost": "127.0.0.1",
+          "rpcport": 30523,
+          "rpcuser": "bitcoin",
+          "rpcpassword": "local321",
+          "zmqpubrawtx": "tcp://127.0.0.1:30633"
+        }
+      ]
+    }
+  }
+```
+
+**Note**: For detailed example configuration see [`regtest/cluster.js`](regtest/cluster.js)
+
 
 ## API Documentation
-These methods are currently only available via directly interfacing with a node:
+Methods are available by directly interfacing with the service:
 
 ```js
 node.services.bitcoind.<methodName>
 ```
 
+### Chain
+
+**Getting Latest Blocks**
+
+```js
+// gives the block hashes within a range of timestamps
+var high = 1460393372; // Mon Apr 11 2016 12:49:25 GMT-0400 (EDT)
+var low = 1460306965; // Mon Apr 10 2016 12:49:25 GMT-0400 (EDT)
+node.services.bitcoind.getBlockHashesByTimestamp(high, low, function(err, blockHashes) {
```

Comment:
Added in: https://github.com/bitpay/bitcore-node/pull/422/commits/950a9d521cef93ac536c965923520c15229bd2e6